util/pkg/vfs/vfs.go

```diff
@@ -59,6 +59,9 @@ type Path interface {
 	// Remove deletes the file
 	Remove() error
 
+	// RemoveAll completely deletes the file (with all its versions and markers)
+	RemoveAll() error
```

Comment:
nit: RemoveAllVersions might be clearer (vs this being a recursive delete or something akin to MkdirAll)